CHANGES.rst

```diff
@@ -1,6 +1,15 @@
-1.1.0 (unreleased)
+1.2.0 (unreleased)
 ------------------
 
+Bug Fixes
+^^^^^^^^^
+

```

Comment:
maybe add mention of fixing pixel coordinate convention

Reply:
I was saying [here](https://github.com/astropy/specreduce/pull/125#issuecomment-1227499628) that the convention hasn't changed, it just wasn't made explicit in `_get_boxcar_weights()` until we got into the weeds of why the overweighting issue occurred. As such, I think mentioning it in the changelog would cause confusion.

Reply:
ah, sorry i missed/forgot that. you are right. leave as-is here.